src/gui/buttons.py

```diff
@@ -62,8 +118,64 @@ def get_new_missions_handler(event: UIEvent) -> UIEventHandler:
     eng.refresh_mission_board()
 
 
-def get_new_missions_button() -> UIFlatButton:
-    btn = UIFlatButton(text="New Missions")
+def get_new_missions_button() -> UITextureButton:
+    main_tex = PixelatedNinePatch(
+        left=1, right=1, bottom=3, top=1, texture=WindowData.buttons[7]
+    )
+    pressed_tex = PixelatedNinePatch(
+        left=1, right=1, bottom=3, top=3, texture=WindowData.buttons[9]
+    )
+    hovered_tex = PixelatedNinePatch(
+        left=1, right=1, bottom=3, top=1, texture=WindowData.buttons[11]

```

Comment:
Maybe we have a `TextureData` class that handles access to textures that are loaded once on the first module import and then are just returned from the in-memory object. Texture loading with the class as currently defined happens every time we import the window_data module (could be wrong about that, worth verifying).

Reply:
Even if arcade is doing some sneaky cache magic under the hood to prevent these re-instantiations of the same textures (which it almost certainly is), it removes a bunch of repeated code